tensorboard/components/tf_card_heading/tf-card-heading-style.html

```diff
@@ -0,0 +1,47 @@
+<!--
+@license
+Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!--
+  Shared stylesheet for figure captions.
+-->
+<dom-module id="tf-card-heading-style">
+  <template>
+    <style>
+
+      /** Horizontal line of labels. */
+      .row {
+        margin-top: -4px;
+        clear: both;
+      }
+
+      /** Piece of text in the figure caption. */
+      .label {
+        float: left;
+        margin-top: 4px;
+        /* TODO(@jart): Use proper width so overflow works. */
+        text-overflow: ellipsis;
+        overflow: hidden;
+      }
+
+      /** Makes label show on the right. */
+      .right {
```

Comment:
It's generally recommended to name classes after their semantic purpose instead of how they will be rendered (ie, positioned on the right side).

That's just a recommendation though, and this case seems rather special because there's no other way to describe something on the right side, so this might be a time to go against that. Maybe `right-side-label`?

Reply:
I kind of like right because when you write the element it says `class="label right"`.